hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconScmTask.java

```diff
@@ -40,8 +46,8 @@ protected ReconScmTask(ReconTaskStatusDao reconTaskStatusDao) {
   private void register() {
     String taskName = getTaskName();
     if (!reconTaskStatusDao.existsById(taskName)) {
-      ReconTaskStatus reconTaskStatusRecord = new ReconTaskStatus(
-          taskName, 0L, 0L);
+      reconTaskStatusRecord = new ReconTaskStatus(
+          taskName, 0L, 0L, 0, 0);
```

Comment:
Already we have. org.hadoop.ozone.recon.schema.tables.pojos.ReconTaskStatus used, as we have created new set of classes, can we check existing logic implementation and combine both as seems we are capturing extra information only.